libpod.conf

```diff
@@ -35,12 +35,6 @@ conmon_env_vars = [
 # CGroup Manager - valid values are "systemd" and "cgroupfs"
 cgroup_manager = "systemd"
 
-# Directory for persistent libpod files (database, etc)
```

Comment:
I think we still want this - what if I want to put it on an entirely separate partition from storage?

Reply:
This is not a large file, so I don't think we need to give the user that kind of flexibility.